### DISCUSSION TITLE
Multiple Bets / Parlay

### DISCUSSION THREAD

**Discussion by weselski** [cloudbet/docs#61] [Ideas]
Hello CloudBet Coding Team, 

I saw in an earlier Discussion Thread that you are working on the Parlay implementation. 
Do you have any news regarding that Topic? 

I would really appreciate it, if you could add this feature. Been waiting for it a long time :)

**Comment by kgravenreuth** [comment]
Hi @weselski 

We don't have any additional updates at the moment for Parlays in our API. You are welcome to bet on our site currently, thank you.

Regards,
Klaus Gravenreuth
Product & Engineering @ Cloudbet

**Comment by jkasztur** [comment]
Hey, checking after 2 more years. Is the parlay betting still on the roadmap?

**Comment by kgravenreuth** [comment]
supporting here now https://www.cloudbet.com/api/?urls.primaryName=Trading%20-%20New 🎉
